### ISSUE TITLE
The necessary of train/validation split

### ISSUE THREAD

**Issue by Dylan-H-Wang** [TengdaHan/DPC#22]
Hi,

Thank you for your code sharing and issuing solving work, it really helps me a lot!

Moreover, I want to ask is that necessary to split data into train/val sets? In other word, is that enough to use the train loss or accuracy to decide the best trained model? Since many unsupervised algorithms e.g. simCLR do not use validation set to decide the best pretrained model. In the [MoCo](https://github.com/facebookresearch/moco) implementation, Kaiming even only uses the last epoch result as explained in [here](https://github.com/facebookresearch/moco/issues/49) and [here](https://github.com/facebookresearch/moco/issues/17).

Thank you!

**Comment by TengdaHan** [comment]
I agree. It's not necessary to split train/val set during self-supervised pretraining. 
But it's still interesting to monitor the val set performance so I kept it here.
